compile.py

```diff
@@ -358,6 +369,8 @@ def write_matrix_tests(requirements, xml_files, adoc_file):
                     _color_=ORANGE_COLOR,
                 )
             )
+            print(f"Test id {test_id} is not present")
```

Comment:
As there is no logging level, you could maybe put "ERROR:" before the message as this is returning an error

Reply:
done